src/types/PaymentIntent.ts

```diff
@@ -166,6 +167,15 @@ export interface SepaParams {
   };
 }
 
+export interface BacsParams {
+  paymentMethodType: 'BacsDebit';
+  paymentMethodData: {
+    accountNumber: string;
+    sortCode: string;
```

Comment:
I believe that, although not **required**, it would be best for us to encrypt this information before transferring it over the Bridge

Reply:
@charliecruzan-stripe  Could you please provide advice on what is the best way to do this? Also, does the new architecture remove the necessity for this?

Reply:
The new architecture would remove the necessity for this, so we should wait for that migration before working on this feature

Reply:
@charliecruzan-stripe Any idea the type of timeframes for this?

Reply:
not at the moment, the new architecture is still very experimental

Reply:
@charliecruzan-stripe Is there anything I can do to help get this over the line?  If the new architecture isn't ready I'm happy to try and add encryption to this PR if that helps.